tools/gulp/tasks/development.ts

```diff
@@ -28,3 +37,10 @@ task(':serve:devapp', serverTask(outDir, true));
 task('serve:devapp', ['build:devapp'], sequenceTask(
   [':serve:devapp', 'material:watch', ':watch:devapp']
 ));
+
+/** Task that copies all vendors into the demo-app package. Allows hosting the app on firebase. */
+task('prepare-deploy:devapp', ['build:devapp'], () => {
```

Comment:
Thought about doing that as well, but it looks like it would require me:
* To specify a firebase project I want to deploy to (not very flexible then - [from here](https://www.npmjs.com/package/firebase-tools#using-as-a-module))
* And there would be the ugly `goOffline()` stuff (hope they improve it at some point..)

Do we have a firebase project we always want to use? If so I can use that one (but not flexible I think)